packages/eslint-config-mc-app/index.js

```diff
@@ -94,6 +96,10 @@ module.exports = {
     'lines-between-class-members': 0,
     // NOTE: The regular rule does not support do-expressions. The equivalent rule of babel does.
     'no-unused-expressions': 0,
+    ...(hasJsxRuntime() && {
```

Comment:
This is what the React docs do.

![Uploading CleanShot 2020-10-25 at 14.38.15@2x.png…]()